### DISCUSSION TITLE
Ender 3 S1 (Pro) - OPEN SOURCE

### DISCUSSION THREAD

**Discussion by ShixinSun** [CrealityOfficial/Creality_Sonic_Pad#99] [General]
Please publish the current source code for your Ender 3 S1 open source printer.
I would like to point out once again that the Ender 3 S1 firmware is subject to the GPL 3.0 license and therefore the current source code must be made available. 

In addition, you advertise the printer itself as open source, but do not provide 2D drawings or CAD files, 3D models, circuit board designs such as schematics and PCB layouts or current source code for the firmware (display / mainboard).
As the printer is also certified by the OSHWA, the hardware must be published with documentation including all source files and the interfaces must be sufficiently documented.

![1caf1caf874713aee983e10bcb26499c](https://github.com/CrealityOfficial/Creality_Sonic_Pad/assets/158030162/97363cb3-b979-4956-b9bb-17466a0ed4c2)

**Comment by liveboxandy** [comment]
Agree totally. disgusted that a company can misuse Open Source in this manner.
